README.md

```diff
@@ -87,7 +87,7 @@ server.start();
 or extend the provided HTTP server and override any methods you need:
 ```js
 // my-custom-express-server.js
-const ExpressHTTPServer = require('fastboot-app-server/lib/express-http-server');
+const ExpressHTTPServer = require('fastboot-app-server/src/express-http-server');
```

Comment:
In our case [`app.listen()`](https://github.com/ember-fastboot/fastboot-app-server/blob/master/src/express-http-server.js#L57) only listens on the IPv6 interface so that the server is unreachable via IPv4 which is a problem. As we cannot change the host via an env var (and @rwjblue had closed #8 which suggested to add that) we'll have to extend `ExpressHTTPServer` and override `serve`. Of course we could build our own server but we'd very much like to avoid that just to change the host.